docs/metrics.md

```diff
@@ -243,6 +243,9 @@ Histogram of VM phase transitions duration from deletion time in seconds. Type:
 ### kubevirt_vmi_phase_transition_time_seconds
 Histogram of VM phase transitions duration between different phases in seconds. Type: Histogram.
 
+### kubevirt_vmi_status_addresses
+The addresses of a VirtualMachineInstance. Type: Gauge.
```

Comment:
maybe consider also add labels info:

"The networking addresses of a VirtualMachineInstance. This includes IP and MAC addresses along with the names of the corresponding host and VMI network interfaces. The 'info_source' label indicates whether the data was sourced from the domain, guest-agent or multus-status"

Reply:
we are following the kube_node_status address so we will reduce the info

Reply:
@machadovilaca  I agree with Aviv about adding more info to the description. This is important for the users and also later in for documentation and AI.

Reply:
should i reference the internal IP, external IP, and hostname or only internal IP at this point?

Reply:
Only what is reported now , the internal IP. If we add more we will update the description.

Reply:
updated then

Reply:
I don't see it yet

Reply:
@machadovilaca Hi, I still can't see an update here for some reason.